catalogue/src/main/kotlin/com/tidal/sdk/catalogue/generated/apis/AlbumJSONAPI.kt

```diff
@@ -0,0 +1,137 @@
+package com.tidal.sdk.catalogue.generated.apis
+
+import com.tidal.sdk.catalogue.generated.infrastructure.CollectionFormats.*
+import retrofit2.http.*
+import retrofit2.Response
+import okhttp3.RequestBody
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+import com.tidal.sdk.catalogue.generated.models.AlbumDataDocument
+import com.tidal.sdk.catalogue.generated.models.AlbumRelationshipsDocument
+import com.tidal.sdk.catalogue.generated.models.AlbumsDataDocument
+import com.tidal.sdk.catalogue.generated.models.ErrorDocument
+
+interface AlbumJSONAPI {
```

Comment:
FYI not reviewing the generated part but I like the method signature. Good parts:
* returning "Response" instead of the direct object
* Suspend funs!